frontend/app-development/features/processEditor/ProcessEditor.tsx

```diff
@@ -52,30 +65,32 @@ export const ProcessEditor = (): React.ReactElement => {
     formData.append('content', new Blob([xml]), 'process.bpmn');
     formData.append('metadata', JSON.stringify(metaData));
 
-    bpmnMutation.mutate(
+    mutateBpmn(
```

Comment:
Is it possible to add tests here as commented by Codecov? 😄

Reply:
To be honest I dont know how to test that 😅
It is very difficult to trigger it since it is triggered by changing things in the bpmn xml modeler from bpmn-js, and it is also difficult to mock it from app-development since we do not have direct access to the bpmnApiContextProvider which holds the function. 
So I might need some help to find a good way to do it, if we want to implement it.

But the test I added for the useBpmnEditor is checking that the callback functions on the bpmnApiContextProvider is triggered on event changes inside the modeler, so we do cover a lot from that one at least 🫣

Reply:
aah, okey. That makes sense 😄 
Great to hear that you have tried and added the one test! 🚀